src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php

```diff
@@ -8,7 +8,7 @@
 {
     const ATTRIBUTE_MESSAGE = 'message';
     /**
-     * @return TagOperation|null
+     * @return AbstractTag|null
```

Comment:
Actually, the returned object should be a TagOperation object as specified by the call
```php
$this->getStrictParent(WsdlDocument::TAG_OPERATION);
```
So I would prefer to let it as it was. it's more accurate.